### PR TITLE
Feat/add eterna design

### LINKED ISSUE #13: Integrate https://bootstrapmade.com/demo/Eterna/ into website

### LINKED ISSUE #14: Create a home page type with carousel
Make the carousel editable via Piranha: https://github.com/PiranhaCMS/piranha.core/issues/623
Page structure:
Carousel (should have the option to add background image, title text, read more link(optional)
Products - enumerable (title, text, icon choosable from FA icons) - eternal cards section 1
Free text - title, html
Contact form

### LINKED ISSUE #15: Add hardcoded bottom menu according to eterna template
Structure:
USEFUL LINKS
[Home]
[About us]
[TEAM product suite]
[Privacy policy](https://bootstrapmade.com/demo/templates/Eterna/#)
OUR SERVICES
[Managed services]
[Web Development]
[Professional services]

CONTACT US
1585 Broadway
New York, NY, 10036, 
United States

Email: hello@team.io

### LINKED ISSUE #16: Add a professional services type page with enumerable services
Services should be cards such as in the "blog" design of Eterna.
Service structure: Image(optional), header, text. Filters of the blog design are not needed.  Read more link pointing to full content page.

### LINKED ISSUE #17: Add a products type page containing TAU EAM products, in accordance to "portfolio" section.
Enumerable products properties: Heading, short text, description text, image, read more link. 
Heading and text should be clickable simulating the plus button, visualizing a carousel with all available products. The carousel should contain the full description text. The thumbnail should have heading and short text. No filtering is needed.

### LINKED ISSUE #18: Add a managed services page type
Design Eterna - services card
Page properties: On top - CMS configurable widgets
Services as cards
Read more URL (optional) - to make the service card clickable

### LINKED ISSUE #19: Add a contacts page
Structure - same as eterna's template, without a map

### LINKED ISSUE #20: Add logo to newly implemented template
```<?xml version="1.0" encoding="UTF-8"?>
<!-- Created with Inkscape (http://www.inkscape.org/) -->
<svg width="62.429mm" height="20.052mm" version="1.1" viewBox="0 0 62.429 20.052" xmlns="http://www.w3.org/2000/svg">
 <g transform="translate(-25.477 -78.548)" display="none">
  <rect x="20.501" y="72.216" width="72.999" height="32.184" fill="#eaa314" stroke-width=".26458"/>
  <g fill="#815300">
   <path transform="matrix(.26458 0 0 .26458 -1.3229 .52917)" d="m118.03 334.98c5.1246-9e-3 9.6651 0.0608 9.6651 0.0608l-2e-3 8.4251s-7.9914-0.03-9.6803 7e-3c-4.7617-1.3176-4.8999-7.4748 0.017-8.4928z" stroke-width=".52093" style="shape-inside:url(#rect4457);white-space:pre"/>
   <path transform="matrix(.26458 0 0 .26458 -1.3229 .52917)" d="m127.66 329.91s-12.358 0.0939-13.504-0.0248c-5.2018-0.8813-5.2018-7.8813-0.23034-8.3562 2.3476-0.19483 13.768-0.13669 13.768-0.13669z" stroke-width=".60541" style="shape-inside:url(#rect4457);white-space:pre"/>
   <path transform="matrix(.26458 0 0 .26458 -1.3229 .52917)" d="m127.54 316.75s-13.751-0.12291-18.412-0.0563c-4.794-0.73707-5.101-7.6981 0.27489-8.2053 4.4436 0.0587 18.181-0.0233 18.181-0.0233z" stroke-width=".71754" style="shape-inside:url(#rect4457);white-space:pre"/>
   <path transform="matrix(.26458 0 0 .26458 -1.3229 .52917)" d="m130.41 348.76 0.0927 7.2958 15.524-0.0391z" stroke-width=".41982" style="shape-inside:url(#rect4457);white-space:pre"/>
  </g>
 </g>
 <g transform="translate(-25.477 -78.548)">
  <path transform="matrix(.26458 0 0 .26458 -1.3229 .52917)" d="m118.03 334.98c5.1246-9e-3 9.6651 0.0608 9.6651 0.0608l-2e-3 8.4251s-7.9914-0.03-9.6803 7e-3c-4.7617-1.3176-4.8999-7.4748 0.017-8.4928z" fill="#815300" stroke-width=".52093" style="shape-inside:url(#rect4457);white-space:pre"/>
  <g fill="#545454">
   <path transform="matrix(.26458 0 0 .26458 -1.3229 .52917)" d="m127.66 329.91s-12.358 0.0939-13.504-0.0248c-5.2018-0.8813-5.2018-7.8813-0.23034-8.3562 2.3476-0.19483 13.768-0.13669 13.768-0.13669z" stroke-width=".60541" style="shape-inside:url(#rect4457);white-space:pre"/>
   <path transform="matrix(.26458 0 0 .26458 -1.3229 .52917)" d="m127.54 316.75s-13.751-0.12291-18.412-0.0563c-4.794-0.73707-5.101-7.6981 0.27489-8.2053 4.4436 0.0587 18.181-0.0233 18.181-0.0233z" stroke-width=".71754" style="shape-inside:url(#rect4457);white-space:pre"/>
   <path transform="matrix(.26458 0 0 .26458 -1.3229 .52917)" d="m118.03 334.98c5.1246-9e-3 9.6651 0.0608 9.6651 0.0608l-2e-3 8.4251s-7.9914-0.03-9.6803 7e-3c-4.7617-1.3176-4.8999-7.4748 0.017-8.4928z" stroke-width=".52093" style="shape-inside:url(#rect4457);white-space:pre"/>
   <path d="m32.61 92.518-3.99e-4 2.2291s-1.1472-0.01648-1.4833-0.0067c-0.9476-0.3486-0.96825-1.9345 0.01027-2.2039 0 0 0.16939-0.0026 1.4734-0.01858z" stroke-width=".11953" style="shape-inside:url(#rect4457);white-space:pre"/>
   <path d="m33.584 92.518 3.99e-4 2.2291s1.1472-0.01648 1.4833-0.0067c0.9476-0.3486 0.96825-1.9345-0.01027-2.2039 0 0-0.16939-0.0026-1.4734-0.01858z" stroke-width=".11953" style="shape-inside:url(#rect4457);white-space:pre"/>
   <a transform="translate(-1.3229 .52917)">
    <g transform="scale(.26458)" style="shape-inside:url(#rect4457);white-space:pre" aria-label="TEAM">
     <g fill="#545454">
      <path d="m146 304s-37.143-0.0392-41.205-0.0528c-4.5143-0.83853-4.8988-8.5407 0.20495-8.9472 16.876-0.10349 29.944 0.0242 40.818 0.11071h0.0417c5.1406 0.88925 5.1406 7.8892 0.14055 8.8892z" stroke-width="1.046"/>
      <path d="m160.24 299.63 0.125 52.625c0 2.2936 1.875 4.2049 4.1667 4.2049l35.583-0.0425h0.0417c2.2917 0 4.1667-1.9113 4.1667-4.2474s-1.875-4.2474-4.1667-4.2474h-0.0417l-31.417 0.0425-0.0417-17.966 16.297 0.0261c2.8617 0.12669 4.435-2.5018 4.1942-4.5744-0.26648-2.2927-1.3498-3.8615-4.2724-3.8621l-16.26-0.1692-0.0417-17.414 31.542 0.0425h0.0417c2.2917 0 4.1667-1.9113 4.1667-4.2474s-1.875-4.2474-4.1667-4.2474h-0.0417l-35.708-0.0425c-2.2917 0-4.1667 1.8688-4.1667 4.1199z" stroke-width="1.0096"/>
      <path d="m236.73 297.37-22.875 53.25c-0.20833 0.45833-0.375 1.2083-0.375 1.7083 0 2.2917 1.875 4.1667 4.1667 4.1667 1.5417 0 3.25-1.1667 3.8333-2.5833l7.2917-16.917 23.75 0.33333 7.125 16.542c0.58333 1.4167 2.3333 2.5417 3.875 2.5417 2.2917 0 4.1667-1.875 4.1667-4.1667 0-0.5-0.16666-1.25-0.37499-1.7083l-22.917-53.167c-0.58334-1.375-2.3333-2.5-3.8333-2.5-1.5417 0-3.25 1.125-3.8333 2.5zm3.8333 12.208 8.3333 19.333-16.542-0.20834z"/>
     </g>
     <path d="m287 351.39v-0.0417l-0.125-37.208 18.375 30.042c0.83334 1.2917 2.125 2 3.7083 2 1.5 0 2.875-1.125 3.5-2.125l16.375-29.75 0.099 48.642s-219.79 0.0378-224.03 0.028c-4.2705 0.49309-3.8468 6.722 0.0995 7.0205 2.1432 0.051 204.71 0.57203 221.67 0.66259 4.3786-0.0687 7.1397-0.87403 9.209-3.4104 1.8043-2.2114 1.3014-8.4628 1.3014-8.4628l-0.0158-59.355c-6.6e-4 -2.4583-2.0833-4.1667-4.4583-4.1667-1.4583 0-2.625 0.83333-3.375 2.0833l-20.625 36.417-22.292-36.375c-0.75-1.5417-2.2083-2.3333-4.125-2.3333-1.25 0-2.25 0.625-2.875 1.625-0.5 0.79167-0.91667 1.5-0.91667 2.25 0 0.29167 0 0.375 0.0417 0.75l0.125 51.708c0.33825 6.6073 8.037 6.1354 8.3334 4e-5z" fill="#e96b56"/>
    </g>
   </a>
  </g>
  <text transform="scale(.26458)" fill="#000000" font-family="sans-serif" font-size="40px" style="line-height:1.25;shape-inside:url(#rect42107);white-space:pre" xml:space="preserve"/>
 </g>
 <g transform="translate(-25.477 -78.548)" display="none" fill="#815300">
  <rect x="26.968" y="83.155" width="7.7029" height="2.391" ry="1.1955" stroke-width=".28565"/>
  <rect x="28.531" y="87.711" width="7.7389" height="2.391" ry="1.1955" stroke-width=".28631"/>
  <rect x="29.925" y="92.505" width="7.7029" height="2.391" ry="1.1955" stroke-width=".28565"/>
  <rect x="30.605" y="81.197" width="2.2266" height="13.575" ry="0" display="inline" stroke-width=".27976"/>
 </g>
</svg>
```

### PR DESCRIPTION
**Pull request by mkyordanov**
- Applied Eterna design
- Created Home page
- Created Services page
- Created Managed Services page
- Created Contact page
- Created Products page
- Added footer
- Added logo

### CODE REVIEW

File: Controllers/CmsController.cs [hunk 1]
```diff
@@ -68,7 +69,107 @@ public async Task<IActionResult> Page(Guid id, bool draft = false)
                 return Unauthorized();
             }
         }
+        
+        /// <summary>
+        /// Gets the home page with the given id.
+        /// </summary>
+        /// <param name="id">The unique page id</param>
+        /// <param name="draft">If a draft is requested</param>
+        [Route("/homepage")]
```
Comment (adautev) [inline Controllers/CmsController.cs]: We want those to be dynamically created by the CMS, not to comply with a certain route. Please test you can add as much dynamic pages to the menu. The default homepage template route should be "/"

File: Controllers/CmsController.cs [hunk 2]
```diff
@@ -68,7 +69,107 @@ public async Task<IActionResult> Page(Guid id, bool draft = false)
...
+        {
+            try
+            {
+                var model = await _loader.GetPageAsync<ServicePage>(id, HttpContext.User, draft);
+
+                return View(model);
+            }
+            catch (UnauthorizedAccessException)
+            {
+                return Unauthorized();
```
Comment (adautev) [inline Controllers/CmsController.cs]: Why should a public page have an unauthorized exception?

File: Controllers/CmsController.cs [hunk 3]
```diff
@@ -68,7 +69,107 @@ public async Task<IActionResult> Page(Guid id, bool draft = false)
...
+            }
+        }
+        
+        /// <summary>
+        /// Gets the contacts page with the given id.
+        /// </summary>
+        /// <param name="id">The unique page id</param>
+        /// <param name="draft">If a draft is requested</param>
+        [Route("/contactpage")]
```
Comment (adautev) [inline Controllers/CmsController.cs]: /contacts

File: Controllers/CmsController.cs [hunk 4]
```diff
@@ -68,7 +69,107 @@ public async Task<IActionResult> Page(Guid id, bool draft = false)
...
+        public async Task<IActionResult> ContactPage(Guid id, bool draft = false)
```
Comment (adautev) [inline Controllers/CmsController.cs:118]: We want those to be dynamically created by the CMS, not to comply with a certain route.  Please test this.

File: Controllers/CmsController.cs [hunk 5]
```diff
@@ -68,7 +69,107 @@ public async Task<IActionResult> Page(Guid id, bool draft = false)
...
+        {
+            try
+            {
+                var model = await _loader.GetPageAsync<ContactPage>(id, HttpContext.User, draft);
+
+                return View(model);
+            }
+            catch (UnauthorizedAccessException)
+            {
+                return Unauthorized();
```
Comment (adautev) [inline Controllers/CmsController.cs]: Same as above, this should be a public page, can't understand the exception handle.

File: Controllers/CmsController.cs [hunk 6]
```diff
@@ -68,7 +69,107 @@ public async Task<IActionResult> Page(Guid id, bool draft = false)
...
+            }
+        }
+        
+        /// <summary>
+        /// Gets the products page with the given id.
+        /// </summary>
+        /// <param name="id">The unique page id</param>
+        /// <param name="draft">If a draft is requested</param>
+        [Route("/productspage")]
```
Comment (adautev) [inline Controllers/CmsController.cs]: /products

File: Controllers/CmsController.cs [hunk 7]
```diff
@@ -68,7 +69,107 @@ public async Task<IActionResult> Page(Guid id, bool draft = false)
...
+        public async Task<IActionResult> ProductsPage(Guid id, bool draft = false)
```
Comment (adautev) [inline Controllers/CmsController.cs:131]: We want those to be dynamically created by the CMS, not to comply with a certain route. Please test we can add as much products as desired.

File: Models/Pages/ContactPage.cs [hunk 8]
```diff
@@ -0,0 +1,13 @@
+namespace TAU.Website.Models.Pages;
+
+using Piranha.AttributeBuilder;
+using Piranha.Extend;
+using Piranha.Models;
+using TAU.Website.Models.Region;
+
+[PageType(Title = "Contact page")]
+[ContentTypeRoute(Title = "ContactPage", Route = "/contactpage")]
```
Comment (adautev) [inline Models/Pages/ContactPage.cs]: /contacts

File: Controllers/CmsController.cs (hunk 2 above)
Comment (mkyordanov) [reply Controllers/CmsController.cs]: It should not, removed

File: Controllers/CmsController.cs (hunk 1 above)
Comment (mkyordanov) [reply Controllers/CmsController.cs]: Many page of the same type can be created. The route is updated. There is a middleware that translates the page`s "slug" to the route and id, so this is not the route that is displayed.

File: Controllers/CmsController.cs (hunk 4 above)
Comment (mkyordanov) [reply Controllers/CmsController.cs:118]: Many page of the same type can be created. The route is updated. There is a middleware that translates the page`s "slug" to the route and id, so this is not the route that is displayed.

File: Controllers/CmsController.cs (hunk 7 above)
Comment (mkyordanov) [reply Controllers/CmsController.cs:131]: Many page of the same type can be created. The route is updated. There is a middleware that translates the page`s "slug" to the route and id, so this is not the route that is displayed.

**Pull request merged**